platform/viewer/src/lib/DICOMSR/dataExchange.js

```diff
@@ -0,0 +1,33 @@
+import OHIF from '@ohif/core';
+import { retrieveMeasurementFromSR, stowSRFromMeasurements } from './handleSR';
+import { getLatestSRSeries } from './srUtils';
+
+export const retrieveMeasurements = () => {
+  console.info('[DICOMSR] retrieveMeasurements');
+
+  const latestSeries = getLatestSRSeries();
+
+  if (!latestSeries) return Promise.resolve({});
+
+  return retrieveMeasurementFromSR(latestSeries);
+};
+
+export const storeMeasurements = (measurementData, server) => {
```

Comment:
When this is called with this patch, I'm finding that the server parameter here is really an object with timepointIds in it. The if check on the server type below always seems to fail and never attempt the stowSRFromMeasurements() below.

Reply:
Yes, this server here is not right, its actually a filter that we are not using right now. 
Forget to remove.